packages/app/src/components/app.tsx

```diff
@@ -15,6 +16,9 @@ flex-direction: column;
 
 export const App: React.FC = () => {
   useMessagePong();
+  useEffect(() => {
+    (window as any).__APP_VERSION__ = version;
```

Comment:
Will this effect server rendered apps? Do we need to do the elaborate global look up?

Reply:
this should not because it's in the app project. the one in connect has a window check already

Reply:
Well, this is in the authenticator, which is never server rendered!

The goal for this global variable if mainly for extension users. If you're using `https://app.blockstack.org`, then we know you're using the latest. It's also used later in tracking, which only happens client side.